packages/camera/camera_android/lib/src/android_camera.dart

```diff
@@ -356,221 +281,102 @@ class AndroidCamera extends CameraPlatform {
 
   @override
   Future<void> setFlashMode(int cameraId, FlashMode mode) =>
-      _channel.invokeMethod<void>(
-        'setFlashMode',
-        <String, dynamic>{
-          'cameraId': cameraId,
-          'mode': _serializeFlashMode(mode),
-        },
-      );
+      _hostApi.setFlashMode(cameraId, flashModeToPlatform(mode));
 
   @override
   Future<void> setExposureMode(int cameraId, ExposureMode mode) =>
-      _channel.invokeMethod<void>(
-        'setExposureMode',
-        <String, dynamic>{
-          'cameraId': cameraId,
-          'mode': serializeExposureMode(mode),
-        },
-      );
+      _hostApi.setExposureMode(cameraId, exposureModeToPlatform(mode));
 
   @override
   Future<void> setExposurePoint(int cameraId, Point<double>? point) {
     assert(point == null || point.x >= 0 && point.x <= 1);
     assert(point == null || point.y >= 0 && point.y <= 1);
 
-    return _channel.invokeMethod<void>(
-      'setExposurePoint',
-      <String, dynamic>{
-        'cameraId': cameraId,
-        'reset': point == null,
-        'x': point?.x,
-        'y': point?.y,
-      },
-    );
+    return _hostApi.setExposurePoint(cameraId, pointToPlatform(point));
   }
 
   @override
   Future<double> getMinExposureOffset(int cameraId) async {
-    final double? minExposureOffset = await _channel.invokeMethod<double>(
-      'getMinExposureOffset',
-      <String, dynamic>{'cameraId': cameraId},
-    );
-
-    return minExposureOffset!;
+    final double minExposureOffset =
+        await _hostApi.getMinExposureOffset(cameraId);
+    return minExposureOffset;
```

Comment:
Nit: here and in the several methods below you can just `return _hostApi.[...]` directly instead of having the variable.